src/app/page.tsx

```diff
@@ -4,15 +4,33 @@ import Form from '@/components/ui/Form';
 import { useRouter } from 'next/navigation';
 import { getBattlePower } from '@/app/calc/ScouterCalculator';
 import styles from '@/styles/scouterText.module.css';
-import { useState, useEffect } from 'react';
-import { ScouterViewer } from '@/components/ui/ScouterViewer';
+import { useState, useEffect, useRef } from 'react';
 
 export default function Home() {
   const router = useRouter();
+  const bgmRef = useRef<HTMLAudioElement | null>(null);
   const [titleAnimated, setTitleAnimated] = useState(false);
   const [subtitleAnimated, setSubtitleAnimated] = useState(false);
   const [showForm, setShowForm] = useState(false);
   const [hasError, setHasError] = useState(false);
+  const [soundEnabled, setSoundEnabled] = useState(true);
+
+  // BGM再生
+  useEffect(() => {
+    bgmRef.current = new Audio('/scouter.mp3');
+
+    if (bgmRef.current) {
+      bgmRef.current.loop = true;
+      bgmRef.current.volume = 0.5;
```

Comment:
[imho]
完全に好みです😂
もう半分で良いかもです
```suggestion
      bgmRef.current.volume = 0.25;
```